stack_overfull/frontend/src/components/NavigationBar/index.jsx

```diff
@@ -73,7 +73,7 @@ export class NavigationBar extends React.Component {
             </Link>
           </div>
           <div className="navbar__search">
-            <SearchBar />
+            <SearchBar/>
```

Comment:
you should always leave a space after the component name. It's a name convention for react.